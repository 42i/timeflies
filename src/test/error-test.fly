wp a
unknown-instruction ha ha ha
off 1
day 2012-09-01 8 10
- a.b 2
- a; duration missing!
- a r; bad duration!
day 2012-09-02, 8, 10
day 2012-09-03, 8 10
day 2012-09-04 10 aa
day 2012-09-05 bb 12
day 2012-09-06 8, 10
day 2012-09-07 8 10 12
must-hours mon=4 5
must-hours MON=5
must-hours mon=gogo
must-hours mon..thu=5
must-hours mon..mon=1
must-hours tue..mon=5
must-hours aaa..fri=3
must-hours mon..fri=yayy
another-unknown-instruction ha ha ha
import error-test.fly
day 2012-09-08 8 10
off wow