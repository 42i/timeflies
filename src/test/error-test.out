error-test.fly:3: ERROR : no current day for instruction "off 1".
error-test.fly:5: ERROR : invalid activity work package "a.b".
error-test.fly:6: ERROR : an activity must have a work package and a duration.
error-test.fly:7: ERROR : invalid activity duration "r".
error-test.fly:8: ERROR : weird instruction "8".
error-test.fly:8: ERROR : weird instruction "10".
error-test.fly:9: ERROR : weird instruction "8 10".
error-test.fly:10: ERROR : bad end time argument "aa" in day spec.
error-test.fly:11: ERROR : bad start time argument "bb" in day spec.
error-test.fly:12: ERROR : unexpected day argument list: "2012-09-06 8".
error-test.fly:12: ERROR : weird instruction "10".
error-test.fly:13: ERROR : unexpected day argument list: "2012-09-07 8 10 12".
error-test.fly:14: ERROR : bad must-hours argument "5".
error-test.fly:15: ERROR : unknown day "MON".
error-test.fly:16: ERROR : bad time argument "gogo" for day "mon".
